jaxrs-api/src/main/java/jakarta/ws/rs/core/Form.java

```diff
@@ -93,4 +93,19 @@ public Form param(final String name, final String value) {
     public MultivaluedMap<String, String> asMap() {
         return parameters;
     }
+
+    /**
+     * Compare for equality
+     * 
+     * @param object the object to compare to.
+     * @return {@code true}, if the object is a {@code Form} with the same form parameters, {@code false} otherwise.
+     */
+    @Override
+    public boolean equals(Object object) {
```

Comment:
I do believe that when `equals` is being implemented, the `hashCode` should be implemented as well.

Reply:
Indeed, the JavaDocs say:
>Note that it is generally necessary to override the hashCode method whenever this method is overridden, so as to maintain the general contract for the hashCode method, which states that equal objects must have equal hash codes.